Documentation/policy.rst

```diff
@@ -763,21 +763,21 @@ endpoint with the label ``id.http`` on port 80:
 
     $ cilium policy trace -s id.curl -d id.httpd --dport 80
     Tracing From: [container:id.curl] => To: [container:id.httpd] Ports: [80/any]
-    * Rule 2 {"matchLabels":{"any:id.httpd":""}}: match
```

Comment:
What is the specific order of the rules?

As I understand it, there's the literal order inside of the internal data structures, but that doesn't have any impact on the application of the rules, and it could be random. The policy should still work the same - it's an unordered whitelist, not an ordered list of priority.

Users can't fetch or insert rules based on this number - the only identifying piece that users can specify is the rule.Labels. It would make more sense to me if we printed the labels (which could be empty) here rather than this iteration number.

Reply:
Right, makes sense.